core/src/test/java/io/questdb/griffin/engine/functions/cast/CastTest.java

```diff
@@ -4727,12 +4727,12 @@ public void testStrToLong256Sort() throws Exception {
                         "\n" +
                         "\n" +
                         "\n" +
+                        "0x99193c2e0a9e76da695f8ae33a2cc2aa529d71aba0f6fec5172a489c48c26926\n" +
                         "0x123455\n" +
                         "0x123455\n" +
                         "0x123455\n" +
                         "0x123455\n" +
-                        "0x8802ff90\n" +
-                        "0x172a489c48c2692600000000529d71ab695f8ae33a2cc2aa99193c2e0a9e76da\n",
+                        "0x8802ff90\n",
```

Comment:
why there is a difference, is it existing bug fixed?

Reply:
existing bug fixed. The test produces a long256 from a symbol "A", then the select statement produces a string representation of the long256 "B" which is not equals to "A". Hence all the refactoring around Numbers/Long256-cast.